submissions/DawoduRahmon/HTML/index.html

```diff
@@ -10,8 +10,9 @@
         <h1>RILOGAD LIMITED</h1>
         <ul>
             <a href="#">Home</a>
+            <a href="file:///C:/Users/User/Desktop/Dufuna-CodeCamp20/submissions/DawoduRahmon/html/product%20page.html" target="blank">Product</a>
```

Comment:
First, remove the space between `product page`. Rename it to `product-page`.

After that, simple do `product-page.html`